editoast/src/main.rs

```diff
@@ -81,6 +81,7 @@ async fn main() {
 async fn run() -> Result<(), Box<dyn Error + Send + Sync>> {
     let client = Client::parse();
     let pg_config = client.postgres_config;
+    let db_pool = Data::new(get_pool(pg_config.url()?, pg_config.pool_size));
```

Comment:
Some commands do not require a db_pool (osm, openapi, some search commands, maybe more). Match against these variants before creating the connection pool.